app/screens/create_or_edit_channel/channel_info_form.tsx

```diff
@@ -0,0 +1,345 @@
+    const placeholderHeader = formatMessage({id: t('channel_modal.headerEx'), defaultMessage: 'Use Markdown to format header text'});
+
+    const makePrivateLabel = formatMessage({id: t('channel_modal.makePrivate.label'), defaultMessage: 'Make Private'});
+    const makePrivateDescription = formatMessage({id: t('channel_modal.makePrivate.description'), defaultMessage: 'When a channel is set to private, only invited team members can access and participate in that channel.'});
+
+    const displayHeaderOnly = channelType === General.DM_CHANNEL || channelType === General.GM_CHANNEL;
+    const showSelector = !displayHeaderOnly && !editing;
+
+    const isPrivate = type === General.PRIVATE_CHANNEL;
+
+    const handlePress = () => {
+        const chtype = isPrivate ? General.OPEN_CHANNEL : General.PRIVATE_CHANNEL;
+        onTypeChange(chtype);
+    };
+
+    const blur = useCallback(() => {
+        nameInput.current?.blur();
+        purposeInput.current?.blur();
+        headerInput.current?.blur();
+        scrollViewRef.current?.scrollToPosition(0, 0, true);
+    }, []);
+
+    const onHeaderLayout = useCallback(({nativeEvent}: LayoutChangeEvent) => {
+        setHeaderPosition(nativeEvent.layout.y);
+    }, []);
+
+    const scrollHeaderToTop = useCallback(() => {
+        if (scrollViewRef?.current) {
+            scrollViewRef.current?.scrollToPosition(0, headerPosition);
+        }
+    }, []);
+
+    const onKeyboardDidShow = useCallback((frames: any) => {
+        setKeyBoardVisible(true);
+        setKeyboardHeight(frames.endCoordinates.height);
+    }, [scrollHeaderToTop]);
+
+    const onKeyboardDidHide = useCallback(() => {
+        setKeyBoardVisible(false);
+        setKeyboardHeight(0);
+    }, []);
+
+    const onScroll = useCallback((e: NativeSyntheticEvent<NativeScrollEvent>) => {
+        const pos = e.nativeEvent.contentOffset.y;
+        if (updateScrollTimeout.current) {
+            clearTimeout(updateScrollTimeout.current);
+        }
+        updateScrollTimeout.current = setTimeout(() => {
+            setScrollPosition(pos);
+            updateScrollTimeout.current = undefined;
+        }, 200);
+    }, []);
+
+    if (saving) {
+        return (
+            <View style={styles.container}>
+                <StatusBar/>
+                <Loading
+                    style={styles.loading}
+                />
+            </View>
+        );
+    }
+
+    let displayError;
+    if (error) {
+        displayError = (
+            <SafeAreaView
+                edges={['bottom', 'left', 'right']}
+                style={styles.errorContainer}
+            >
+                <View style={styles.errorWrapper}>
+                    <ErrorText
+                        testID='edit_channel_info.error.text'
+                        error={error}
+                    />
+                </View>
+            </SafeAreaView>
+        );
+    }
+
+    const autoComplete = (
+        <Autocomplete
```

Comment:
Since the Autocomplete component is not displayed conditionally, you can call it directly in the render method.